src/lake/tests/init/.gitignore

```
@@ -5,6 +5,9 @@
 /hello-exe
 /lean-data
 /123-hello
+# the follow test directory has unicode names that
+# can confuse git, hence the possibly redundant ignores
 /«A.B».«C.D»
+/«A-B»-«C-D»
```

```suggestion
/«A-B»-«C-D»
```

As such, it is fixing it should be enough, there is likely no need for the redundancy.